src/MSTGQLStore.ts

```diff
@@ -112,17 +112,22 @@ export const MSTGQLStore = types
     function subscribe<T = any>(
       query: string | DocumentNode,
       variables?: any,
-      onData?: (item: T) => void
+      onData?: (item: T) => void,
+      onError: (error: Error) => void = error => {
```

Comment:
Thanks for the quick reply. So it is optional because I am supplying a default `onError` here and the default `onError` is just a simple function that throws the passed error. If you prefer I can make it optional and use a check inline and throw if the function isnt defined.

Reply:
yup, you are correct, I totally glanced over this line!